vncviewer/ClientConnection.cpp

```diff
@@ -77,8 +78,8 @@ extern "C" {
 #else
 #define MAX_ENCODINGS (LASTENCODING+52)
 #endif
-#define VWR_WND_CLASS_NAME _T("VNCviewer")
-#define VWR_WND_CLASS_NAME_VIEWER _T("VNCviewerwindow")
+#define VWR_WND_CLASS_NAME _T("ultravnc-viewer")
+#define VWR_WND_CLASS_NAME_VIEWER _T("ultravnc-viewer-window")

```

Comment:
Other software connect to the viewer like this
vb. sample token from the net
WinHandle = FindWindow("vncviewer", vbNullString)

If we change the window name, they all need to change the name.
Even when the name is bad, we are not going to touch it.